it/uk/gov/hmrc/claimvatenrolmentfrontend/views/CaptureBusinessPostcodeViewTests.scala

```diff
@@ -59,5 +63,17 @@ trait CaptureBusinessPostcodeViewTests {
       doc.getSubmitButton.first.text mustBe Base.continue
     }
   }
+  def testCaptureVatRegistrationPostcodeRedirectViewTests(result: => WSResponse,
+                                                          authStub: => StubMapping): Unit = {
+
+    lazy val doc: Document = {
+      authStub
+      Jsoup.parse(result.body)
+    }
+
+    "correctly display the error summary" in {
+      doc.getErrorSummaryTitle.text mustBe Base.Error.title
```

Comment:
The error summary should also have a body and there should also be a field error (look in CaptureBox5FigureViewTests). You'll need to update MessageLookup with the error messages and then you should be able to use them in your tests.  You can find the messages you need in the messages file in the conf folder
Also, could you add another one of these for an invalid postcode (testCaptureBusinessPostcodeInvalidErrorViewTests) because the error message is different for a missing postcode and an invalid one